src/mpyl/build.py

```diff
@@ -175,7 +175,14 @@ def run_mpyl(
                 console.log(f"Exception during build execution: {exc}")
                 console.print_exception()
 
-            console.print(Markdown(run_result_to_markdown(run_result)))
+            try:
```

Comment:
I don't think this is the best solution, it just hides the actual problem? We also never had these markdown issues, so it must be something new or related to this pr which we should fix

Reply:
Agree, I'll work on it

Reply:
Addressed in : https://github.com/Vandebron/mpyl/pull/294/commits/abed1d80695b25ee47189b790a4ce2fd2fb4f96d